akka-docs/src/main/paradox/cluster-sharding.md

```diff
@@ -334,6 +334,13 @@ then supposed to stop itself. Incoming messages will be buffered by the `Shard`
 between reception of `Passivate` and termination of the entity. Such buffered messages
 are thereafter delivered to a new incarnation of the entity.
 
+## Automatic Passivation
+
+The entities can be configured to be automatically passivated if they haven't received
+a message for a while using the `akka.cluster.sharding.passivate-idle-entity-after` setting,
+or by explicitly setting `ClusterShardingSettings.passivateIdleEntityAfter` to a suitable
+time to keep the actor alive. By default automatic passivation is disabled. 
```

Comment:
do we need to mention that messages that are not sent via sharding are not counted, e.g. scheduled messages to `self`?

Reply:
Good point, I'll add that.